src/Duration.elm

```diff
@@ -24,12 +24,12 @@ elapsedSecondsFromDurations durations =
 updateLatest : Posix -> List Duration -> List Duration
 updateLatest moment durations =
     let
-        ( ( begin, _ ), befores ) =
+        ( ( begin, _ ), olders ) =
```

Comment:
I can't remember why this have multiple items, just using 2 size tuple is not enough? 🤔